src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConstants.java

```diff
@@ -34,19 +29,6 @@
   public static final String FLOAT_TYPE = "float";
   public static final String DOUBLE_TYPE = "double";
   public static final String STRING_TYPE = "string";
+  public static final String TEXT_TYPE = "text";
```

Comment:
ES also has a `keyword` type. Should we support it?

Reply:
I can add it now as a placeholder.